x-pack/plugins/observability_solution/observability_onboarding/public/application/quickstart_flows/firehose/create_stack_command_snippet.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import {
+  EuiAccordion,
+  EuiCodeBlock,
+  EuiLink,
+  EuiSpacer,
+  EuiText,
+  useGeneratedHtmlId,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import {
+  FIREHOSE_CLOUDFORMATION_STACK_NAME,
+  FIREHOSE_STREAM_NAME,
+} from '../../../../common/aws_firehose';
+import { CopyToClipboardButton } from '../shared/copy_to_clipboard_button';
+import { buildCreateStackCommand, buildStackStatusCommand } from './utils';
+
+interface Props {
+  encodedApiKey: string;
+  onboardingId: string;
+  elasticsearchUrl: string;
+  templateUrl: string;
+  isCopyPrimaryAction: boolean;
+}
+
+export function CreateStackCommandSnippet({
+  encodedApiKey,
+  elasticsearchUrl,
+  templateUrl,
+  isCopyPrimaryAction,
+}: Props) {
+  const stackStatusAccordionId = useGeneratedHtmlId({ prefix: 'stackStatusAccordion' });
+  const createStackCommand = buildCreateStackCommand({
+    templateUrl,
+    stackName: FIREHOSE_CLOUDFORMATION_STACK_NAME,
+    streamName: FIREHOSE_STREAM_NAME,
+    encodedApiKey,
+    elasticsearchUrl,
+  });
+  const stackStatusCommand = buildStackStatusCommand({
+    stackName: FIREHOSE_CLOUDFORMATION_STACK_NAME,
+  });
+
+  return (
+    <>
+      <EuiText>
+        <p>
+          <FormattedMessage
+            id="xpack.observability_onboarding.firehosePanel.createFirehoseStreamDescription"
+            defaultMessage="Run the command bellow in your terminal where you have {awsCLIInstallGuideLink} configured. The command will create a CloudFormation stack that includes a Firehose delivery, backup S3 bucket, CloudWatch subscription filter and metrics stream along with required IAM roles."
+            values={{
+              awsCLIInstallGuideLink: (
+                <EuiLink
+                  data-test-subj="observabilityOnboardingFirehosePanelAwsInstallGuideLink"
+                  href="https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html"
+                  external
+                  target="_blank"
+                >
+                  {i18n.translate(
+                    'xpack.observability_onboarding.firehosePanel.awsCLIInstallGuideLinkLabel',
+                    { defaultMessage: 'AWS CLI' }
+                  )}
+                </EuiLink>
+              ),
+            }}
+          />
+        </p>
+      </EuiText>
+
+      <EuiSpacer />
+
+      <EuiCodeBlock language="text" paddingSize="m" fontSize="m">
```

Comment:
Besides the command, should we also provide a link to the quick start template link? https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-console-create-stacks-quick-create-links.html

AFAIK it should work mostly the same

Reply:
Added this as a [follow up task](https://github.com/elastic/kibana/issues/190799). @patpscal is working on the updated designs, most probably there will be a flow for AWS UI.